### PR TITLE
Replaces scan button with fab

### LINKED ISSUE #167: Shift qr scan button to FAB
Currently QR scan button is on attendees page at bottom which takes space. It should be in the app bar of `EventContainerActivity` as it is event specific action.

I am working on it.

### PR DESCRIPTION
**Pull request by mejariamol**
fixes #167 

also renames `activity_event_container.xml` to `fragment_event_container.xml` to avoid confusions.

screenshot_2017-06-12-15-59-09
screenshot_2017-06-12-15-59-29

### CODE REVIEW

**Comment by mejariamol** [comment]
@iamareebjamal @rohanagarwal94 @Shailesh351 @Ronak-59 @mahikaw please review

**Comment by mejariamol** [comment]
@iamareebjamal done :+1:

File: app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ public void showProgressBar(boolean show) {
 
     @Override
     public void showScanButton(boolean show) {
-        ViewUtils.showView(btnBarCodeScanner, show);
+        ViewUtils.showView(fabBarcodeScanner, show);
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java]: Chane this to `fab.show()` and `fab.hide()`

Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java]: thanks. done :+1:

**Pull request merged**